pegasus/sites.v3/code.org/public/educate/csd-alt.md.erb

```diff
@@ -269,7 +269,7 @@ In addition to computer access, you'll need:
 * Choose from one of the following sets of materials for collaborative activities:
   * Option 1 - [Aluminum Boats](https://curriculum.code.org/csd-20/unit1/1/): Aluminum foil and containers that can hold water
   * Option 2 - [Newspaper Tower](https://curriculum.code.org/csd-20/unit1/9/): A set of newspapers and several tape dispensers
-  * Option 3 - [Spaghetti Bridge](https://curriculum.code.org/csd-20/unit1/10/): Several boxes of spaghetti and several glue guns 
+  * Option 3 - [Spaghetti Bridge](https://curriculum.code.org/csd-20/unit1/10/): Several boxes of spaghetti and several glue guns

```

Comment:
There are a bunch of links here that still go to Curriculum Builder for CSD 20-21. Wondering if this page needs to be updated or is old and should be deprecated

Reply:
I don't see any reference csd-alt, and this page hadn't been updated since this time last year prior to today. this should probably be deleted. @kelbyhawn perhaps you can advise us on whether we can just delete this page, or whether we should let you go through your own process for doing it?

in any case Dani, I think this page looks unused so I'm not going to make any other updates to it in this PR.

Reply:
Thanks for checking! This page is in my queue for deletion so I can take care of it 👍 For context, here's my Pruning PR: https://github.com/code-dot-org/code-dot-org/pull/47035 — it's still in progress, but hoping to get everything deleted in one fell swoop in the next couple weeks.

Reply:
Love it!!!

<img width="410" alt="Screen Shot 2022-06-30 at 9 51 42 AM" src="https://user-images.githubusercontent.com/8001765/176733999-81e5a006-3612-448f-9205-694e0cbeed04.png">